scripts/mirror.js

```diff
@@ -350,7 +358,8 @@ const bumpOpera = (originalData, sourceData, source) => {
   if (typeof sourceData.version_added === 'string') {
     newData.version_added = getMatchingBrowserVersion(
       'opera',
-      browsers[source].releases[sourceData.version_added],
+      source,
```

Comment:
Since you're already renaming variables, might it be a good opportunity to rename `source` to `sourceBrowser` here as well?